Cython/Compiler/Nodes.py

```diff
@@ -3877,6 +3885,14 @@ def generate_argument_values_setup_code(self, args, code):
     def generate_keyword_unpacking_code(self, min_positional_args, max_positional_args,
                                         has_fixed_positional_count,
                                         has_kw_only_args, all_args, argtuple_error_label, code):
+        # First we count how many arguments must be passed as positional
```

Comment:
I think we know this already, right? There's an instance attribute for it.

Reply:
This instance attribute is sometimes different from this value, so I didn't change this.  See the longer comment below for details.